README.md

```diff
@@ -55,6 +55,12 @@ This action only runs on Linux, as it uses [podman](https://github.com/container
     <td>Yes</td>
     <td>Password, encrypted password, or access token with which to authenticate to the registry.</td>
   </tr>
+
+   <tr>
+    <td>tls-verify</td>
+    <td>No</td>
+    <td>Require HTTPS and verify certificates when contacting registry</td>
```

Comment:
Can you put just a little more detail on this and in action.yml

`Skip TLS certificate verification when contacting registry.`

It should also end in a period.